lib/core/utils/is-xhtml.js

```diff
@@ -0,0 +1,16 @@
+
+/**
+ * Determines if a document node is XHTML
+ * @method isXHTML
+ * @memberof axe.utils
+ * @instance
+ * @param {Node} doc a document node
+ * @return {Boolean}
+ */
+axe.utils.isXHTML = function (doc) {
+	'use strict';
+	if (!doc.createElement) {
+		return false;
+	}
+	return doc.createElement('A').localName === 'A';
```

Comment:
What does this do on JSDOM? We should make sure we don't accidentally treat JSDOM as XHTML.

Reply:
Yes, confirmed to work on JSDOM.